.gitignore

```diff
@@ -136,3 +136,6 @@ dmypy.json
 
 # Cython debug symbols
 cython_debug/
+
+# Ignore the downloaded PDFs
+*.pdf
```

Comment:
In `git`, you can use a `.gitignore` file to ensure certain types of files and folder don't accidentally get into your repository.

If I run `git status` without the change above AND I've downloaded pdfs, 
then I will see that it would be very easy to accidentally `git add` the pdfs to the repository. 

This could be desirable in some situations, but I don't think you intended that. 
It seem like you just want to download the pdfs.